opencog/benchmark/atomspace_bm.cc

```diff
@@ -25,6 +27,8 @@ int main(int argc, char** argv)
      "          \t(default: 100000)\n"
      "-r <int>  \tLooping count; how many times a python/scheme operation is looped\n"
      "          \t(default: 1)\n"
+     "-R <int>  \tUse specific randomseed; useful for benchmark comparisons\n"
+     "          \t(default: time(NULL))\n"
```

Comment:
Yes, there is. I was seeing a spread of about 4% to 5% at times for the timing of the getHandlesOfType call that dropped to about 1% once I started using any particular fixed randomseed.

It's also useful for stress testing to have different values at times as bugs sometimes show up for rare occurrences. That's also why I'm now printing the value so you can go back and repeat a sequence if you run into a problem.